tests/conftest.py

```diff
@@ -20,6 +20,7 @@
 
```

Comment:
is this needed?

Reply:
by 'this' I mean the whole conftest ... and ppx ...

Reply:
Some of it is needed. You are right that there are some unused functions. I'll remove those and the imports that use them. Thanks for the catch!